dashi/src/lib/components/DashiDropdown.tsx

```diff
@@ -8,6 +8,7 @@ import { type PropertyChangeHandler } from "@/lib/types/model/event";
 
 export interface DashiDropdownProps extends Omit<DropdownState, "type"> {
   onPropertyChange: PropertyChangeHandler;
+  panelIndex: number;
```

Comment:
The term `panel` does not exist is the API, the `lib` part. This is relevant fort the demo only.

Also, I don't see why this is needed in every component now.